razzle.config.js

```diff
@@ -40,7 +40,6 @@ module.exports = {
             // externals means files not generated by webpack
             // an alternative would be html-webpack-plugin
             caches: 'all',
-            externals: ['/'],
```

Comment:
This has been removed as now that Simorgh returns non-200 responses, this value was causing the service worker to fail as nothing is served over `/`.

Reply:
This is interesting. What about static assets such as svgs? They presumably just are no longer cached by the sw?